core/lib/Thelia/Type/EnumListType.php

```diff
@@ -36,6 +36,17 @@ public function addValue($value)
         }
     }
 
+    /**
+     * @param array $values
+     * @since 2.3.0
+     */
+    public function addValues($values = array())
```

Comment:
IMO if you don't want to use type hinting here, you have to test if `$values` is an `array` or an instance of `Traversable` and throw an exception if it's not the case